src/test/test_gridcoin.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-2018 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef GRIDCOIN_TEST_TEST_GRIDCOIN_H
+#define GRIDCOIN_TEST_TEST_GRIDCOIN_H
+
+#include <main.h>
+#include <random.h>
+#include <wallet/wallet.h>
+
+extern uint256 insecure_rand_seed;
+extern FastRandomContext insecure_rand_ctx;
+
+static inline void SeedInsecureRand(bool fDeterministic = false)
+{
+    if (fDeterministic) {
+        insecure_rand_seed = uint256();
+    } else {
+        insecure_rand_seed = GetRandHash();
+    }
+    insecure_rand_ctx = FastRandomContext(insecure_rand_seed);
+}
```

Comment:
Are we aiming for deterministic "replay" style randomness or truly random here? Because this function is actually never called. Since it is not called, the actual initialization is already done in test_gridcoin.cpp at line 27 and 28, and since there the seed is randomly generated, it is equivalent to reinitializing with SeedInsureRand(false). Also not sure why we are not using g_insecure_rand_ctx and initializing a new global object.

Reply:
Whoops, I removed the deterministic randomness (Bitcoin uses it to ensure tests are deterministic) from the .cpp file but missed the bit in the header.  I'll removing that block real quick